### ISSUE TITLE
Add a language menu chooser

### ISSUE THREAD

**Issue by BoFFire** [mastodon/mastodon-android#173]
Hi,

Some minority languages such as `kabyle` language needs to be choosen manually as it is not fully available in Android Core. We need a language chooser.
Thank you 🥰

**Comment by gunchleoc** [comment]
Yes, please! Tusky has this feature, so it's technically doable. 🙂
